packages/docusaurus/src/client/exports/withBaseUrl.js

```diff
@@ -7,7 +7,8 @@
 
 import useDocusaurusContext from './useDocusaurusContext';
 
-function withBaseUrl(url: string): string {
+export default function withBaseUrl(url) {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
eslint is complaining React Hook "useDocusaurusContext" is called in function "withBaseUrl" which is neither a React function component or a custom React Hook function react-hooks/rules-of-hooks

By right we should name this useBaseUrl because its a hook

Reply:
This is bad, we + migrated sites need to rename so many things. Thankfully still in beta. We should rename it asap.